spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -44,18 +44,42 @@ describe 'orifice-calculator-viewmodel-spec', ->
       expect(viewModel.pipeID().length).toEqual 6
 
     it 'should have default value of "1.939\'\' XS, Sch 80, Sch 80S"', ->
-      expect(viewModel.selectedPipeID()).toEqual '1.939\'\' XS, Sch 80, Sch 80S'
+      expect(viewModel.selectedPipeID()).toEqual OPL.OrificeCalculator.Config.Dictionaries.PipeID.one_nine_inch
```

Comment:
I reckon it might have been me the first one to write it this way, but JS variables use camel case:
`OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch`